python/ray/serve/_private/application_state.py

```diff
@@ -186,6 +263,13 @@ def get_app_status(self, name: str) -> ApplicationStatusInfo:
             )
         return self._application_states[name].get_application_status_info()
 
+    def list_app_status(self) -> Dict[str, ApplicationStatusInfo]:
+        """Return a list of applications"""
```

Comment:
Could you update this docstring? The function is returning a dictionary instead of a list now.